controller/src/main/java/org/jboss/as/controller/transform/ResourceTransformationContextImpl.java

```diff
@@ -412,7 +405,7 @@ public TransformersLogger getLogger() {
         private final ImmutableManagementResourceRegistration registration;
 
         OriginalModel(Resource original, RunningMode mode, ProcessType type, TransformationTarget target, ImmutableManagementResourceRegistration registration) {
-            this.original = original;
+            this.original = original.clone();
```

Comment:
This largely defeats the point of https://github.com/wildfly/wildfly-core/pull/1102/files#diff-89ed7c94b256f549e7a6789336b54404R135 . I'm not saying it's wrong, I'm just a little sad. :(

I think I'll keep the Stage.DOMAIN operation attachment thing though. It at least saves the cost of doing a full OC.readResourceFromRoot each time, and limits the expense to just a Resource.clone().